common/persistence/cassandra/cassandraHelpers.go

```diff
@@ -26,11 +26,10 @@ import (
 	"os"
 	"strings"
 
-	"github.com/uber/cadence/common/auth"
-
 	"github.com/gocql/gocql"
 	log "github.com/sirupsen/logrus"
 
+	"github.com/uber/cadence/common/auth"
```

Comment:
Curious is there any IDE plugin sorting imports automatically?

Reply:
I think current if there's no empty line between those imports, `make fmt` will automatically group them in a standard way.